src/lib/bind/accessors.html

```diff
@@ -172,14 +172,35 @@
       // be interrogating the prototype here
       // TODO(sorvell): we want to avoid using `getPropertyInfo` here, but
       // this requires more data in `_propertyInfo`
-      var info = model.getPropertyInfo && model.getPropertyInfo(property);
-      if (info && info.readOnly) {
+      var info = model.getPropertyInfo && model.getPropertyInfo(property) || {};
+      if (info.readOnly) {
         // Computed properties are read-only (no property setter), but also don't
         // need a private setter since they should not be called by the user
         if (!info.computed) {
           model['_set' + this.upper(property)] = setter;
         }
       } else {
+        if (info.set) {
+          var setFn = model[info.set];
+
+          if (setFn) {
+            var processing = false;
+
+            setter = function(value) {
+              // When setting the value, effects are triggered. In order to
+              // prevent stackoverflows, do not re-process the updated value
+              if (!processing) {
+                processing = true;
```

Comment:
Don't you actually need this lock/flag only b/c your setFn's in the tests are actually not idempotent?